backend/drm/drm.c

```diff
@@ -1284,7 +1285,22 @@ void scan_drm_connectors(struct wlr_drm_backend *drm) {
 		wl_list_for_each(c, &drm->outputs, link) {
 			index++;
 			if (c->id == drm_conn->connector_id) {
-				wlr_conn = c;
+				struct wlr_output edid_holder;
+				size_t edid_len = 0;
+				uint8_t *edid = get_drm_prop_blob(drm->fd, c->id,
+					c->props.edid, &edid_len);
+
+				parse_edid(&edid_holder, edid_len, edid);
+				free(edid);
+
+				if (strcmp(c->output.make, edid_holder.make) == 0
+					&& strcmp(c->output.model, edid_holder.make) == 0
```

Comment:
That ```make``` seems to be a typo.

Also, you compare make/model/serial again and not the edid blob id as mentioned in the commit message.

Reply:
Yep, I also fixed up the commit message, thanks for the review :+1: